### PR TITLE
Update login form UI & create shared css

### LINKED ISSUE #6: Finalize UI for Login Flow (Login Page and Forgot Password)

### PR DESCRIPTION
**Pull request by samarthdave**
Closes #6 
### I added Bootstrap 4 CSS from CDN
...but I guess we can build Bootstrap from scratch. However, even with the proxy, this CSS import fails:
```
<link rel="stylesheet" href="/auth/static/css/style.css" />
```

Edit: Nevermind, I fixed it by removing the "/auth" from the path:
```
/static/css/style.css
```

### CODE REVIEW

**Comment by samarthdave** [comment]
I think I'm also going to shrink the login form to be a bit smaller and similar to the [Bootstrap 4 Sign in Example](https://getbootstrap.com/docs/4.0/examples/sign-in/)

**Comment by samarthdave** [comment]
## :construction: Still a construction site right now :construction:
Cleaning up the login page & solved the proxy problem. Turns out running ```dev:sass``` doesn't do it. You have to manually save for it to trigger a generation of the final file.

**Comment by samarthdave** [comment]
Question: Are we identifying Gatekeeper as it's own entity? I went ahead and included TD logo for the login to retain the flow of the app.

**Comment by samarthdave** [comment]
Okay so I spent the last hour trying to get floating labels to work...
[https://getbootstrap.com/docs/4.0/examples/floating-labels](https://getbootstrap.com/docs/4.0/examples/floating-labels)
It didn't work for a couple of reasons. Namely, some calculation bug referenced here:
https://github.com/twbs/bootstrap/issues/27629

**Comment by geooot** [comment]
@samarthdave are you using localhost:4000 because the styles load fine for me?

**Comment by phulsechinmay** [comment]
image

@samarthdave Was playing around with how form inputs look and came up with this. What do you think?

**Comment by geooot** [comment]
@samarthdave @phulsechinmay  
- Remember to put a link saying "Don't have an account?" which leads to the signup page
- @danielabreo should be kept in the loop on what UI decisions you make as he's working on the sign-up page

**Comment by samarthdave** [comment]
I think that looks good. I more prefer [this](https://getbootstrap.com/docs/4.0/examples/floating-labels/). I tried to get it to work with bootstrap but it was being a little difficult.

I ended up getting the styles to work @geooot

**Comment by samarthdave** [comment]
On second thought, that looks fine but can I pull it to try it locally or on my desktop & mobile?
You might have to clone the branch and make another branch for me to test the UI.

Or we can just merge what we have and make a separate issue for the input card @phulsechinmay ? Even though issue #6 is for UI, I think we should share the same CSS for the signup and login...

**Comment by samarthdave** [comment]
@phulsechinmay I think we shouldn't have to scroll (though my laptop isn't that tall) to see the footer copyright section. Should we add some horizontal content like this (view in incognito if you're already signed into Devpost):
[https://devpost.com/users/login](https://devpost.com/users/login)

**Comment by phulsechinmay** [comment]
@samarthdave Yeah could always move the social login boxes to the side, or reduce the margin on top and play around with the heights / margins of stuff

**Comment by geooot** [comment]
After messing around I think this is a good direction:
image

Notes that should be addressed:
- Change the body from `height: 100%` to `min-height:100%`, on devices with short height the UI gets cut off
- Make sure to use either "Sign in" or "Login" as text but don't use them both
- Add the sentence under the title to indicate that after logging in they will be redirected to something else. In the future this will be customizable by the backend
- I don't really like the current sign in button (its a bit tacky). I reverted the styles to normal bootstrap ones and used btn-lg to make it bigger
- Add the create an account link
- I used `border-top-left-radius: 0.25rem` to get border radius on one specific corner (kinda like the bootstrap example)
- I made the buttons `btn-block` and the social buttons `btn-outline-primary`

**Comment by samarthdave** [comment]
ok wowow that looks wayyy better than the changes I made. Can you push that?

**Comment by geooot** [comment]
@samarthdave I just did it in devtools not code sorry!

**Comment by samarthdave** [comment]
I'll replicate it then! One sec

**Comment by samarthdave** [comment]
I still need to update the social login buttons & "login" vs "sign in"

**Comment by samarthdave** [comment]
I thought deleting the comments would make it easier to read our comments but I guess not whoops.

Here's what it looks like now @geooot 
image

**Comment by geooot** [comment]
@samarthdave Can we change "Submit" to "Log In"

**Comment by samarthdave** [comment]
Kept the "i" lowercase since I think it looks a bit cleaner. We should be good other than making the style.css used in both "login" & "signup" page. I'll reach out to @danielabreo

File: views/login/index.ejs [hunk 1]
```diff
@@ -1,54 +1,67 @@
...
+        <input type="checkbox" value="remember-me"> Remember me
+      </label>
+    </div> -->
+
+    <div id="submit-btn-div"" class="text-center text-left">
+      <button type="submit" class="btn btn-primary btn-block btn-lg mx-auto">Log in</button>
     </div>
 
+    <div class="text-center mt-3">
+      <a href="/auth/signup">Create an account</a>
```
Comment (geooot) [inline views/login/index.ejs:43]:
```suggestion
      <a href="/auth/signup?r=<%= locals.redirectLink %>">Create an account</a>
```

**Pull request merged**